Options.php

```diff
@@ -59,8 +59,9 @@ public function get_key() {
 	 * @return mixed Whatever is in those fields.
 	 */
 	public function get( $field = null, $default = null ) {
+		$option_array = get_option( $this->key, array() );
+		$option_array = gettype($option_array) != "array" ? [] : $option_array;
 		$data = array_merge( $this->defaults, get_option( $this->key, array() ) );
```

Comment:
You might need to use `$option_array` on this line. Otherwise the PR doesn't do anything.